helpdesk-message-proxy-service/src/main/java/org/symphonyoss/symphony/bots/helpdesk/messageproxy/service/TicketService.java

```diff
@@ -157,4 +164,27 @@ public void sendIdleMessageToAgentStreamId(SymMessage message) {
       LOGGER.error("Could not send ticket message to agent stream ID: ", e);
     }
   }
+
+  public void sendMessageWhenRoomCreationFails(SymMessage symMessage) {
```

Comment:
Please include javadoc on new methods and classes.

Reply:
Done.